.circleci/config.yml

```diff
@@ -1,6 +1,9 @@
 version: 2.1
+on:
+  pull_request:
+
 jobs:
-  build_frontend:
+  build_frontend_and_deploy:
```

Comment:
while we are at it, let's splt this into:

- `frontend_tests` with test and lint
- `forntend_build_and_deploy`

do the same split for the api

thanks :)